back/middlewares/game-guard.js

```diff
@@ -0,0 +1,12 @@
+const { game } = require('../models/game');
+
+const gameGuard = (req, res, next) => {
+  const gamePlayers = game.players.length;
+  if (gamePlayers <= 0) {
```

Comment:
gamePlayersCount === 0. Length cannot be negative

Reply:
Done.